lib/loaders/total.js

```diff
@@ -20,7 +20,7 @@ export const total = (path, options = {}) => {
 export const totalLoader = httpLoader(total);
 
 const load = (path, options = {}) =>
-  totalLoader.load(path, options)
+  totalLoader.load({ path, options })
```

Comment:
What I found was an example in the DataLoader docs here https://github.com/facebook/dataloader#using-with-graphql, which passes an array:

```js
queryLoader.load([
  'SELECT toID FROM friends WHERE fromID=? LIMIT ?', user.id, first
])
```

I figured I’d just give passing an arbitrary single argument a try and its worked. However, now I’m thinking that maybe it’s silently breaking anything? Like caching or something? I’m not familiar atm with what DataLoader does under the hood.